setup.py

```diff
@@ -1,62 +0,0 @@
-"""setup module."""
-
-import re
-from pathlib import Path
-
-from setuptools import find_packages, setup
-
-name = "mse_cli"
-
-version = re.search(
-    r"""(?x)
-    __version__
-    \s=\s
-    \"
-    (?P<number>.*)
-    \"
-    """,
-    Path(f"{name}/__init__.py").read_text(),
-)
-
-setup(
-    name=name,
-    version=version["number"],
-    url="https://cosmian.com",
-    license="MIT",
-    project_urls={
-        "Documentation": "https://docs.cosmian.com",
-        "Source": "https://github.com/Cosmian/mse-cli",
```

Comment:
Can we try to keep these references URL in the toml file? 
Those are displayed in the pypi page.